python/pip_install/tools/wheel_installer/wheel.py

```diff
@@ -228,14 +234,27 @@ class Deps:
     def __init__(
         self,
         name: str,
+        *,
+        requires_dist: Optional[List[str]],
         extras: Optional[Set[str]] = None,
         platforms: Optional[Set[Platform]] = None,
     ):
         self.name: str = Deps._normalize(name)
+        self._platforms: Set[Platform] = platforms or set()
+
+        reqs = sorted(
```

Comment:
nit: Is this sorted call for correctness, determinism, or just being defensive? Please drop a brief comment as to why.